doc/classes/SceneTree.xml

```diff
@@ -263,6 +263,9 @@
 			- 2D and 3D physics will be stopped. This includes signals and collision detection.
 			- [method Node._process], [method Node._physics_process] and [method Node._input] will not be called anymore in nodes.
 		</member>
+		<member name="physics_interpolation" type="bool" setter="set_physics_interpolation_enabled" getter="is_physics_interpolation_enabled" default="false">
+			Although physics interpolation would normally be globally turned on and off using [member ProjectSettings.physics/common/physics_interpolation], this property allows control over interpolation at runtime.
```

Comment:
This needs to be entirely reworded but I have no specific suggestion at the moment ([working on something else](https://github.com/godotengine/godot/pull/87334)).
Something more akin to how the equivalent Project Setting is described, which then ends in like "This corresponds to the ... project setting".